packages/playground/lib/src/dynamic.css

```diff
@@ -0,0 +1,4 @@
+@import 'https://cdn.jsdelivr.net/npm/@mdi/font@5.9.55/css/materialdesignicons.min.css';
```

Comment:
Wonder if we should use a different URL or a postcss plugin to avoid relying on this as CI would run and fetch this everytime. But maybe it's not a big deal.

Reply:
https://github.com/vitejs/vite/blob/main/packages/playground/lib/index.dist.html
`./lib/style.css` is not used so it won't be loaded indeed.
I feel it is better to be done but I don't know a good way.

Reply:
I think we can further improve this again if needed, don't want to stall the PR :)